src/PrestaShopBundle/Twig/Component/NavBar.php

```diff
@@ -28,14 +28,144 @@
 
 namespace PrestaShopBundle\Twig\Component;
 
+use Configuration;
+use Context;
+use Dispatcher;
+use Link;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Routing\Exception\RouteNotFoundException;
 use Symfony\UX\TwigComponent\Attribute\AsTwigComponent;
+use Tab;
+use Tools;
 
 #[AsTwigComponent(template: '@PrestaShop/Admin/Component/Layout/nav_bar.html.twig')]
 class NavBar
 {
-    public string $toggleNavigationUrl;
-    public string $defaultTabLink;
-    public string $psVersion;
-    public bool $collapseMenu;
-    public array $tabs;
+    private Link $link;
+
+    public function __construct(
+        private readonly LegacyContext $context,
+        private readonly LoggerInterface $logger,
+    ) {
+        $isSecureMode = Tools::usingSecureMode() && Configuration::get('PS_SSL_ENABLED');
+        $protocol = $isSecureMode ? 'https://' : 'http://';
+        $this->link = new Link($protocol, $protocol);
+    }
+
+    public function getToggleNavigationUrl(): string
+    {
+        return $this->link->getAdminLink('AdminEmployees', true, [], [
```

Comment:
You can use directly the context like this:
```suggestion
        return $this->context->getAdminLink('AdminEmployees', true, [], [
```
(And maybe, you can avoid the link use on this component!)